test/components/views/rooms/BasicMessageComposer-test.tsx

```diff
@@ -35,30 +35,43 @@ describe("BasicMessageComposer", () => {
 
     it("should allow a user to paste a URL without it being mangled", () => {
         const model = new EditorModel([], pc, renderer);
+        const client: MatrixClient = MatrixClientPeg.get();
 
-        const wrapper = render(model);
+        const roomId = "!1234567890:domain";
+        const userId = client.getUserId();
```

Comment:
Recently learned about

```suggestion
        const userId = client.getSafeUserId();
```

Will let you not do a lot of the type safety  afterwards